### ISSUE TITLE
Launching Controller from Visual Studio is not Working

### ISSUE THREAD

**Issue by DavidMansolino** [cyberbotics/webots#1179]
**Describe the Bug**
Using the wizard it is possible to create a controller which then compiles fine with Visual Studio but then when trying to start it from Visual Studio (as an extern controller) it fails to start the controller:
![Capture](https://user-images.githubusercontent.com/2461619/70615024-e925ad80-1c0b-11ea-8125-9d4d9e1a456f.PNG)

I am almost sure that this was working with the Webots R2019b.

**Comment by DavidMansolino** [comment]
I confirm this is working fine with Webots R2019b rev1

**Comment by DavidMansolino** [comment]
I checked the diff between the VS file generated from R2019rev1 and R2020a and here is the only difference (which seems to make sense):
![1](https://user-images.githubusercontent.com/2461619/70974457-1361ea00-20a8-11ea-9b30-14b8aed84e1f.PNG)

**Comment by DavidMansolino** [comment]
Actually with R2019b rev1 it was working only when Visual Studio was opened by Webots but not anymore when opening manually Visual Studio.
Therefore I think it can be explained by the fact that in R2019b rev1 Webots exectuable and libController were in the same folder and therefore when starting Visual Studio from Webots, libController was in the PATH (which is not the case in R2020a anymore since libController and Webots  executable are not in the same folder anymore).
In that case it is still a bug, but almost not anymore a regression.

**Comment by DavidMansolino** [comment]
Adding ``WEBOTS_HOME/lib/controller`` to the PATH fixes the problem, so probably as a quick fix we can document it (and in a second step fix our VS template).

**Comment by DavidMansolino** [comment]
Adding a ``*.vcxproj.user`` file with this content to the template fixes the problem:
````
<?xml version="1.0" encoding="utf-8"?>
<Project ToolsVersion="Current" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|x64'">
    <LocalDebuggerEnvironment>PATH=%WEBOTS_HOME%/lib/controller;%PATH%</LocalDebuggerEnvironment>
    <DebuggerFlavor>WindowsLocalDebugger</DebuggerFlavor>
  </PropertyGroup>
</Project>
````

**Comment by DavidMansolino** [comment]
Fixed in #1209
